### PR TITLE
PGEN Updates for AoU [VS-1254]

### PR DESCRIPTION
**Pull request by mcovarr**
Currently based off Kevin's branch, I plan to merge this to ah_var_store after Kevin's branch merges. We should then merge / pick this back to the EchoCallset branch and add an entry for EchoCallset in .dockstore.yml

### CODE REVIEW

File: scripts/variantstore/docs/aou/AOU_DELIVERABLES.md [hunk 1]
```diff
@@ -70,16 +72,22 @@
...
     - This workflow needs to be run with the `filter_set_name` input from `GvsCreateFilterSet` step.
     - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
+1. `GvsExtractCallsetPgenMerged` workflow
+    - Run the `GvsPrepareRangesCallset` workflow first following the steps described above for the `GvsCallsetStatistics` workflow.
+    - Specify the same `call_set_identifier`, `dataset_name`, `project_id`, and `extract_table_prefix` that were used in the `GvsPrepareRangesCallset` run documented above.
+    - Specify the `interval_list` and `interval_weights_bed` appropriate for the PGEN extraction run you are performing. `gs://gvs_quickstart_storage/weights/gvs_full_vet_weights_1kb_padded_orig.bed` is the interval weights BED used for Quickstart. The expectation is that there will be multiple PGEN extraction runs for a variety of interval lists:
+      - [Exome-only](https://broadworkbench.atlassian.net/browse/VS-1279)
+      - [ACAF threshold](https://broadworkbench.atlassian.net/browse/VS-1109)
+      - others?
```
Comment (rsasch) [inline scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:90]: I'm kinda uncomfortable to adding callset-specific tickets to a "general" doc for AoU callsets.

Comment (mcovarr) [reply scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:90]: Happy to remove this as I'm struggling to get documentation on what interval lists we're going to use for PGEN / VCF extract anyway...

Comment (rsasch) [reply scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:90]: You can see that the three deliverables are "ACAF Threshold", "Exome" and "ClinVar" here: https://support.researchallofus.org/hc/en-us/articles/14929793660948-Smaller-Callsets-for-Analyzing-Short-Read-WGS-SNP-Indel-Data-with-Hail-MT-VCF-and-PLINK

Comment (mcovarr) [reply scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:90]: Sweet, thank you!

**Review by rsasch — APPROVED** [review]

**Review by gbggrant — APPROVED** [review]

**Pull request merged**